sql/core/src/test/scala/org/apache/spark/sql/execution/command/DDLSuite.scala

```diff
@@ -1371,7 +1371,7 @@ abstract class DDLSuite extends QueryTest with SQLTestUtils {
       // if (isUsingHiveMetastore) {
       //  assert(storageFormat.properties.get("path") === expected)
       // }
-      assert(storageFormat.locationUri === Some(expected))
+      assert(Some(storageFormat.locationUri.get.getPath) === Some(expected.getPath))
```

Comment:
Do we need to add `Some`?

What is the reason we need to change this test case?

Reply:
![image](https://user-images.githubusercontent.com/5399861/57514224-8c46d980-7342-11e9-9248-2be01171dd54.png)
Add `Some` to support both Hive 1.2 and 2.3.

Reply:
`storageFormat.locationUri.map(_.getPath)` then